src/components/Navigation/NavigationList/NavigationList.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { px } from '../../../utils';
+import classNames from 'classnames';
+import { NavigationItemProps } from '../NavigationItem/NavigationItem';
+import { HeaderContext } from '../../Header/Header';
+
+export type NavigationListProps = React.HTMLAttributes<HTMLElement>;
+
+const NavigationList = ({ children, className }: React.PropsWithChildren<NavigationListProps>) => {
+  const { expanded } = React.useContext(HeaderContext);
+  const largeCtaItems: React.ReactNode[] = [];
+  const smallCtaItems: React.ReactNode[] = [];
+  React.Children.map(
+    children as React.ReactElement<NavigationItemProps>[],
+    (child: React.ReactElement<NavigationItemProps>) => {
+      if (child && child.props.navGroup === 'nav-link-lg') {
+        largeCtaItems.push(child);
+      } else if (child && child.props.navGroup === 'nav-link-sm') {
+        smallCtaItems.push(child);
+      }
+    },
+  );
+
+  return (
+    <ul
+      id="nav-list"
+      data-testid="nav-list"
```

Comment:
I think @davidicus said we need this for aria `role="list"`

Also needs an `aria-expanded`:
https://www.accede-web.com/en/guidelines/rich-interface-components/drop-down-menu/#:~:text=The%20aria%2Dexpanded%20attribute%20must,associated%20sub%2Dmenu%20is%20expanded.

Reply:
I think ul have an implicit role=list